dao-modules/src/lib.rs

```diff
@@ -1,9 +1,12 @@
+mod governance_voting;
 mod owner;
 mod repository;
 mod staking;
 mod token;
 mod whitelist;
 
+pub use governance_voting::voting::VotingId;
+pub use governance_voting::*;
```

Comment:
Don't use `*`. It's good for development, but at the end it's better to have explicit imports.